resources/install/debian/postinst

```diff
@@ -67,7 +67,7 @@ case "$1" in
 
         fi
 
-        OLD_JITSI_CONFIG="/usr/share/jicofo/.sip-communicator/sip-communicator.properties"
+        OLD_JITSI_CONFIG="/etc/jitsi/jicofo/sip-communicator.properties"
```

Comment:
This is not correct as the old config is /usr/share/jicofo/.sip-communicator/sip-communicator.properties, it is used to migrate old config to new one.